capabilityquerier/cmd/main.go

```diff
@@ -51,7 +53,12 @@ func queryEndpoints(message []byte, args *map[string]interface{}) error {
 	urlString := string(message)
 	exportFileWait := viper.GetInt("exportfile_wait")
 
+	thresholdInt := viper.GetInt("pruning_threshold")
+	threshold := strconv.Itoa(thresholdInt)
+	queryInterval := strconv.Itoa(thresholdInt + (2 * viper.GetInt("capquery_qryintvl")))
```

Comment:
i'm curious why queryInterval is calculated as `thresholdInt + (2 * viper.GetInt("capquery_qryintvl"))` here